### PR TITLE
[Misc #19860] [DOC] Update license phrases to SPDX BSD-2-Clause

### PR DESCRIPTION
**Pull request by nobu**
There seem to be many variations on the "2-clause BSDL" these days. The current BSDL file seems to be the old style of the BSD 2 Clause "Simplified" license (BSD-2-Clause), so clarified this and updated it to a more recent version: https://spdx.org/licenses/BSD-2-Clause.html

### CODE REVIEW

File: BSDL [hunk 1]
```diff
@@ -1,22 +1,24 @@
-Copyright (C) 1993-2013 Yukihiro Matsumoto. All rights reserved.
+Copyright (C) 1993-2023 Yukihiro Matsumoto. All rights reserved.
```
Comment (hsbt) [inline BSDL:1]:
```suggestion
Copyright (C) 1993 Yukihiro Matsumoto. All rights reserved.
```

Comment (hsbt) [reply BSDL:1]: End date is not necessary for copyright. We should leave to update it.

**Comment by nurse** [comment]
License needs to be carefully reviewed

**Comment by hsbt** [comment]
This is not only for GitHub metadata. We may need to update the recent version provided by SPDX.

**Comment by hsbt** [comment]
It's not time to update this now.

**Pull request closed without merging**